src/vs/editor/contrib/inlineCompletions/browser/ghostTextWidget.ts

```diff
@@ -74,6 +77,9 @@ export class GhostTextWidget extends Disposable {
 	private readonly replacementDecoration = this._register(new DisposableDecorations(this.editor));
 
 	private update(): void {
+		if (this._ariaStringBuilder.build().length) {
+			return;
+		}
```

Comment:
Why is this private state? I think it makes more sense to allocate this per update call (in a local variable).

Reply:
Yep, makes sense